### DISCUSSION TITLE
Does PaddleOCR have a copyright？

### DISCUSSION THREAD

**Discussion by Sadakooooooo** [PaddlePaddle/PaddleOCR#6964] [Q&A] [answered]
Does PaddleOCR have a copyright？We want to use PaddleOCR for personal use, is PaddleOCR copyrighted?

**Comment by Evezerest** [comment, marked as answer]
Yes, PaddleOCR is available under Apache-license 2.0, which could be used for both personal and commercial purposes

**Reply by Sadakooooooo** [reply]
Thanks for helping me ! I really appreciate it!
